src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplatePhrase/PhrasalTemplatePhrase.styles.ts

```diff
@@ -0,0 +1,67 @@
+import { Accordion, AccordionDetails, AccordionSummary, styled } from '@mui/material';
+
+import { theme, variables } from 'shared/styles';
+
+export const StyledAccordion = styled(Accordion)<{ component?: string }>({
+  background: 'none',
+  border: 'none',
+  boxShadow: `0 -2px 0 -1px ${variables.palette.outline_variant2}`,
+  margin: theme.spacing(0, -3),
+  padding: theme.spacing(0, 3),
+
+  '& [data-toggle]': { transition: variables.transitions.all },
+
+  '&.Mui-expanded': {
+    margin: theme.spacing(0, -3),
+    padding: theme.spacing(0, 3),
+
+    '& [data-toggle]': {
+      transform: 'rotate(-180deg)',
+    },
+  },
+});
+
+StyledAccordion.defaultProps = {
+  component: 'li',
+  square: true,
+  variant: 'outlined',
+};
+
+export const StyledAccordionSummary = styled(AccordionSummary)({
+  margin: 0,
+  minHeight: 'unset',
+  padding: theme.spacing(3, 0),
+
+  '&& .MuiAccordionSummary-content': {
+    margin: 0,
+
+    '&.Mui-expanded': { margin: 0 },
+  },
+});
+
+export const StyledAccordionDetails = styled(AccordionDetails)({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: theme.spacing(2.4),
+  padding: theme.spacing(0, 0, 1.6, 0),
```

Comment:
Each accordion element actually needs to have a full 32 pixels for its bottom padding to have correct spacing with the horizontal divider line when there are multiple phrases:

```suggestion
  padding: theme.spacing(0, 0, 3.2, 0),
```

To support this change, the `ItemOptionContainer` used in `PhrasalTemplate` should have its bottom padding removed (pending my [other suggested change](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1865#discussion_r1700475707) to `ItemOptionContainer`):

```tsx
// PhrasalTemplate.tsx line 48

      <ItemOptionContainer sx={{ mb: 0, pb: 0 }}>
```